react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
The requirement is that this label has 18px text on smaller screens and 14px text on larger screens.  The only other option I can see here is to write a media query here?